packages/desktop-client/src/components/Notifications.js

```diff
@@ -27,7 +27,7 @@ function compileMessage(message, actions, setLoading, onRemove) {
             {parts.map((part, idx) => {
               let match = part.match(/\[([^\]]*)\]\(([^)]*)\)/);
               if (match) {
-                let [_, text, href] = match;
+                let [, text, href] = match;
```

Comment:
I like having the `_` here so it’s clear that a value is being ignored, would it make sense to exclude defining the variable `_` from the lint rule?

Reply:
I don't have a strong opinion here, so went ahead with the suggestion.

https://github.com/actualbudget/actual/pull/421/commits/b3ebff7b59e3f0fe1981791b83d6a08dab3b8a3a